back_end/packages/Microsoft.AspNet.Mvc.5.2.3/Content/Web.config.install.xdt

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0"?>

```

Comment:
Do we need all these files in packages version control?
I'm not great with C# but I assume the back_end/RayTracer/packages.config will specify what we need and people can install it themselves rather than keeping it in git?